docs/atlascli/command/atlas-projects-create.txt

```diff
@@ -75,6 +75,10 @@ Options
      - string
      - false
      - Unique 24-digit string that identifies the Atlas user to be granted the Project Owner role on the specified project. If unspecified, this value defaults to the user ID of the oldest Organization Owner.
+   * - --tag
+     - stringToString
+     - false
+     - List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the project. This value defaults to [].
```

Comment:
I haven't seen this null default before, should there be a default value here? Otherwise, shouldn't this line automatically be omitted?

Reply:
I've checked some other commands which use tags. They do have the sentence "This value defaults to [].", but the weird formatting makes it look like that line doesn't belong to the tag. Example:

>   * - --tag
>     - stringToString
>     - false
>     - List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster.
>
>       Mutually exclusive with --file. This value defaults to [].

https://github.com/mongodb/mongodb-atlas-cli/blob/master/docs/atlascli/command/atlas-clusters-create.txt#L132

In this command there's no `--file` flag to be mutually exclusive with, that's why it's on the same line.
I've checked some other commands which have the `--tag` flag and the behavior seems to be consistent, these are some I've checked:

- [clusters create](https://github.com/mongodb/mongodb-atlas-cli/blob/master/docs/atlascli/command/atlas-clusters-create.txt#L132)
- [clusters update](https://github.com/mongodb/mongodb-atlas-cli/blob/master/docs/atlascli/command/atlas-clusters-update.txt#L106)
- [clusters upgrade](https://github.com/mongodb/mongodb-atlas-cli/blob/master/docs/atlascli/command/atlas-clusters-upgrade.txt#L105)
- [deployments setup](https://github.com/mongodb/mongodb-atlas-cli/blob/master/docs/atlascli/command/atlas-deployments-setup.txt#L121), not mutually exclusive, same like this commands

Same with `stringToString`, it seems to be the same with other commands.

Reply:
Created a PR for the default value: https://github.com/mongodb-labs/cobra2snooty/pull/50